frescobaldi_app/exception.py

```diff
@@ -40,9 +41,17 @@ class ExceptionDialog(QDialog):
     def __init__(self, exctype, excvalue, exctb):
         super(ExceptionDialog, self).__init__()
 
+        # _tbshort is the exception line only (last line of the traceback)
         self._tbshort = ''.join(traceback.format_exception_only(exctype, excvalue))
+        # _tbfull is the full traceback
         tbfull = traceback.format_exception(exctype, excvalue, exctb)
         self._tbfull = ''.join(tbfull)
+        # _tbshorter is the traceback, but truncated to 5 frames max.
+        # We do this because of size limits on what you can pass to GitHub
+        # in a URL.
+        tbshorter = traceback.format_exception(exctype, excvalue, exctb, limit=5)
```

Comment:
nitpick: I fear the wording here is potentially misleading thinking that shorter is even shorter than "short". Could it be called "tbmedium" or perhaps "tblimited"?